upgrade/db/mysql/ezplatform-2.5.latest-to-3.0.0.sql

```diff
@@ -39,3 +39,21 @@ ALTER TABLE `ezeditorialworkflow_markings`
     ADD COLUMN `reviewer_id` INT(11),
     ADD COLUMN `result` TEXT;
 --
+
+-- EZEE-2988: Added availability for schedule hide --
+BEGIN;
```

Comment:
To be consistent within the file (see the first line):
```suggestion
START TRANSACTION;
```